sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -9,110 +9,36 @@ import {
-        .framework_ContentPlacement {
-            border-width: 0 var(--border-width) var(--border-width) 0;
-        }
+    :host([section="framework"]) {
+        grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+        contain: content;
+        overflow: hidden;
     }
 
-    @media screen and (max-width: 900px) {
-        :host {
-            --flow: row;
-        }
-    }
-
-    @media screen and (min-width: 700px) and (max-width: 999px) {
-        .framework_container {
-            grid-template-columns: repeat(2, 1fr);
-        }
-
-        .community_container {
-            grid-template-columns: repeat(3, 1fr);
-        }
-
-        .framework_ContentPlacement:nth-of-type(even) {
-            border-right: none;
-        }
-
-        .framework_ContentPlacement:nth-last-of-type(-n + 2):nth-of-type(odd) {
-            /* The last two items on medium screens have no bottom border */
-            border-bottom: none;
-        }
-    }
-
-    @media screen and (min-width: 1000px) {
-        .framework_container,
-        .community_container {
-            grid-template-columns: repeat(3, 1fr);
-        }
-
-        .framework_ContentPlacement:nth-of-type(3n),
-        .framework_ContentPlacement:first-of-type
-            + .framework_ContentPlacement:last-of-type,
-        .framework_ContentPlacement:only-of-type {
-            border-right: none;
-        }
-
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n):nth-last-of-type(-n + 3), /* first item when only three items are present */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n) ~ .framework_ContentPlacement:nth-last-of-type(-n + 3), /* last three items when full row of three */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n - 1) ~ .framework_ContentPlacement:nth-last-of-type(-n + 2), /* more than one row, last row with only two items */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(3n - 2) ~ .framework_ContentPlacement:last-of-type, /* more than one row, last row with only one item */
-        .framework_ContentPlacement:first-of-type:nth-last-of-type(-n + 2) {
-            /* only two items on a row that should fit three */
-            border-bottom: none;
-        }
-
-        .framework_ContentPlacement:nth-of-type(6) {
-            border: none;
-        }
+    :host([section="community"]) {
+        grid-template-columns: repeat(auto-fit, minmax(225px, 1fr));
     }
 
     /* This creates the color, background, and elevation changes on hover */
-    /* start */
-    .community_container:hover site-content-placement,
-    .feature_container:hover site-feature-card {
+
+    :host([section="community"]:hover) site-content-placement {
         color: var(--neutral-foreground-hint);
     }
-    .community_container:hover site-content-placement:hover,
-    .feature_container:hover site-feature-card:hover {
+
+    :host([section="community"]) site-content-placement:hover {
```

Comment:
I believe that we should  still include feature on hover here. on hover all not hovered elements should be greyed out and currently why are being displayed as such. 
![Screen Shot 2020-06-02 at 12 55 55 PM](https://user-images.githubusercontent.com/48499261/83565311-0f4d4b80-a4d3-11ea-8ff9-5a1790ba35cc.png)